packages/Textarea/src/Textarea.js

```diff
@@ -37,7 +37,7 @@ const defaultProps = {
 };
 
 function Textarea(props) {
-  let textarea = null;
+  let textarea = React.useRef(null);
 
   const resize = () => {
     if (textarea && textarea.style) {
```

Comment:
This now has to use `textarea.current` (also there are assignments to `textarea` elsewhere that need to be changed)

Reply:
Using `textarea.current` on any of these values breaks the resize functionality for me, I'me not even sure that `textarea` needs to be a ref?. Can you let me know whats wrong with it just being a variable of `null`?